samples/bluetooth/mesh/sensor_server/prj.conf

```diff
@@ -65,6 +65,12 @@ CONFIG_BT_MESH_SENSOR_SRV=y
 CONFIG_BT_MESH_SENSOR_SRV_SENSORS_MAX=5
 CONFIG_BT_MESH_SENSOR_USE_LEGACY_SENSOR_VALUE=n
 
+# Increase default delayable message config to avoid insuffisient resources
+# during normal sample operation
+CONFIG_BT_MESH_ACCESS_DELAYABLE_MSG_CHUNK_COUNT=60
+CONFIG_BT_MESH_TX_SEG_MSG_COUNT=2
```

Comment:
Why does it require 2 Tx sar contexts? There are only 2 devices in sample. Two sar transactions to the same address are not allowed.

Reply:
You could be doing Config Server operations (that result in long responses) and simultaneously fetching Sensor_Series_Get. If these two things happen together, sample will start printing errors and models will fail to send second SAR message, even when DSTs are different.

Reply:
Sensor model transactions before or in parallel sensor model configuration? Hm... sounds doubtful.

Reply:
Hmm... Well before increasing this, the following error was triggered from time to time: 
https://github.com/zephyrproject-rtos/zephyr/blob/d651be5d70322e24b8940ae0c3a68a7d552cce38/subsys/bluetooth/mesh/transport.c#L508

If you are correct in your statement, I have no Idea why increasing this makes a difference.

Reply:
However, that might be the case when client received transport ack and started the new sar transaction while server still sends transport acks. Could you set sar configuration to single transport ack (without repeats)?

Reply:
The `BT_MESH_SAR_RX_ACK_RETRANS_COUNT` was set to 0 in the sensor server sample, so I have no idea what is causing this. @alxelax maybe we should make a investigation task/bug report to investigate this further? If you want I can remove the mentioned option from this PR for now, but then ofc it will produce error output from time to time.

Reply:
Ok, leave your improvements `as is`. I guess it is worth to investigate the reason of the issue. I'm Ok if it is done it as a separate task.